src/daemon/wgutils.cpp

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <arpa/inet.h>
```

Comment:
We have the includes following these guidelines:

1. first the internal headers
2. after an empty line, the external headers
3. each block (internal and external headers) alphabetic sorted
4. if there is a corresponding header for the cpp class, that goes on top. In this case, the first internal header is wgutils.h (even if not alphabetic sorted)

So, in this case the order would be:

```
#include "wgutils.h"
#include "daemon.h"
#include "logger.h"
#include "loghandler.h"
#include "wgquickprocess.h"  // source of WG_INTERFACE

#include <arpa/inet.h>
#include <QString>
```

Then, QString is not required, I suspect.